src/state/vaults.ts

```diff
@@ -48,12 +48,12 @@ type VaultHistory = {
 
 const initialState: VaultsState = {
   vaultDetails: {
-    totalValue: 30_425,
+    totalValue: 930_425_857,
     thirtyDayReturnPercent: 0.1474,
     positions: [
-      {
+      ...range(1).map((a) => ({
         asset: { id: 'PEPE', name: 'Pepe' },
-        marketId: 'PEPE-USD',
+        marketId: `PEPE-USD${a}`,
```

Comment:
[q] if we do `range(1)` isn't that the same as just adding a 0? 
also am curious in general:
- where does the totalValue come from?
- why do we want to suffix the marketId

Reply:
I can remove this, it was just for testing purposes. 

Total value is going to come from the indexer. Suffix was because market id is a key/should be unique.